### PR TITLE
Fixed: Queue not always clearing checked items when updated

### LINKED ISSUE #223: Pull Sonarr commit 'Fixed: Queue not always clearing checked items when updated'
Pull https://github.com/Sonarr/Sonarr/commit/bba57bb4345be37096f107d8358322308cba8737

```
git cherry-pick bba57bb4345be37096f107d8358322308cba8737
```

This commit has 1 conflict(s)

You can ask me to `close all`, `open pr here` or `open pr everywhere`

[AB#1187](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/1187)

### PR DESCRIPTION
**Pull request by PearsonFlyer**
(cherry picked from commit f4beb54e8c0ba17d80808f503e0385aced75f81b)

# Conflicts:
#	frontend/src/Activity/Queue/Queue.js

#### Database Migration
NO

#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Todos
- [ ] Tests
- [ ] Wiki Updates

#### Issues Fixed or Closed by this PR

* Fixes #223 
* Fixes [AB#1187](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/1187)

### CODE REVIEW

**Comment by Qstick** [comment]
Already pulled here 8d43d5d7b0c7a39adec48b622cd9f1dcdbb2a42a. Looks like your PR just reorders some properties in line with Sonarr I presume?

**Comment by PearsonFlyer** [comment]
That looks correct.  If you'd rather, we can mark those issues closed and kill this PR. I'm happy either way as long as the issue gets closed.

File: frontend/src/Activity/Queue/Queue.js [hunk 1]
```diff
@@ -139,10 +140,8 @@ class Queue extends Component {
       allUnselected,
       selectedState,
       isConfirmRemoveModalOpen,
-      isPendingSelected,
-      items
+      isPendingSelected
     } = this.state;
-
```
Comment (Qstick) [inline frontend/src/Activity/Queue/Queue.js:145]: Can you add back this line break, then good.

Comment (PearsonFlyer) [reply frontend/src/Activity/Queue/Queue.js:145]: Done.

**Pull request closed without merging**